calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -90,6 +91,29 @@ If a malicious or suspicious program is run within the cluster, it will be repor
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+### Create an exception for suspicious programs
+
+If you have a program running within the cluster and want to negate the generation of any runtime logs for that process, thus negating any Security Events for that process, then you can add an entry into the RuntimeExceptionList of the Runtime Security resource through kubectl :
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: RuntimeSecurity
+metadata:
+  name: default
+spec:
+  runtimeExceptionList:
+    - matching: exact
+      processInvocation: "runc init"
+      namespace: default
+    - matching: regex
+      processInvocation: "\b(shred|rm|mv).*((bash|ash|zsh|fish|fish_read)_history|history)\b"
```

Comment:
If this is an example regex match? I'm not sure at all how this works.

Reply:
yeah this is an example. I'll label the code block like you suggested